conf/CatalogoDatiInformativi.xsd

```diff
@@ -160,7 +160,7 @@
 
   <xs:simpleType name="stImportoMassimoFascia">
     <xs:restriction base="xs:decimal">
-      <xs:enumeration value="5.00"></xs:enumeration>
```

Comment:
La lista delle fasce è stata decisa essere fissa quindiè corretta la restizione. Se il CDI non passa è corretto

Reply:
@furlaniilaria

Reply:
Attualmente, in Prod, ci sono più di 400 PSP che hanno CDI con importi non compresi in questa lista (complice il fatto che, tempo fa, il valore massimo fosse 99999999.99 e non 999999999.99). Tutti questi PSP non verrebbero restituiti nella nodoChiediInformativaPSP, tra questi ci sono quasi tutti gli acquirer presenti sul WISP.